DuckDB.NET.Data/DuckDBAppender.cs

```diff
@@ -0,0 +1,181 @@
+﻿using System;
+#nullable enable
+namespace DuckDB.NET.Data;
+
+public class DuckDBAppender : IDisposable
+{
+	private readonly DuckDBConnection _connection;
+	private readonly string _table;
+	private readonly NET.DuckDBAppender _nativeAppender;
+
+	internal DuckDBAppender(DuckDBConnection connection, string table)
+	{
+		_connection = connection;
+		_table = table;
+
+		NativeMethods.Appender.DuckDBAppenderCreate(connection.NativeConnection, null, table, out _nativeAppender);
+	}
+
+
+	public DuckDBAppenderRow CreateRow()
+	{
+		// https://duckdb.org/docs/api/c/api#duckdb_appender_begin_row
+		// Begin row is a no op. Do not need to call.
+		// NativeMethods.Appender.DuckDBAppenderBeingRow(_connection.NativeConnection)
+
+		return new DuckDBAppenderRow(_nativeAppender);
+	}
+	
+	private void ReleaseUnmanagedResources()
+	{
+		NativeMethods.Appender.DuckDBAppenderFlush(_nativeAppender);
+		NativeMethods.Appender.DuckDBAppenderClose(_nativeAppender);
+	}
+
+	private void Dispose(bool disposing)
+	{
+		ReleaseUnmanagedResources();
+	}
+
+	public void Dispose()
+	{
+		Dispose(true);
+		GC.SuppressFinalize(this);
+	}
+
+	~DuckDBAppender()
+	{
+		Dispose(false);
+	}
+}
+
+public class DuckDBAppenderRow : IDisposable
+{
+	private readonly NET.DuckDBAppender _appender;
+
+	internal DuckDBAppenderRow(NET.DuckDBAppender appender)
+	{
+		_appender = appender;
+	}
+
+	public void AppendValue(bool value)
```

Comment:
I think all `AppendXXX` methods should `return this` to have fluent interface.

Reply:
Fair enough